Documentation/Configuration/Typo3ConfVars/FE.rst

```diff
@@ -65,9 +65,16 @@ $GLOBALS['TYPO3_CONF_VARS']['FE']['compressionLevel']
 
    Determines output compression of FE output. Makes output smaller but
    slows down the page generation depending on the compression level. Requires
-   zlib in your PHP installation. Range 1-9, where 1 is least compression and
-   9 is greatest compression. true as value will set the compression based on
-   the PHP default settings (usually 5). Suggested and most optimal value is 5.
+
+   * zlib in your PHP installation and
```

Comment:
Please indent lists with 2 spaces, the next line with 3 spaces, see https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/CommonPitfalls/Lists.html